### PR TITLE
Try setting the `cmd` key on the Android build Docker spec

### PR DESCRIPTION
**Pull request by rye**
This was a suggestion from a Feb '17 response on the CircleCI discussion board: https://discuss.circleci.com/t/builds-getting-killed-with-vague-message-received-signal-killed/10214/9

This would override the value if it's getting set weirdly.  Let's see what happens.

Closes #2170.

### CODE REVIEW

**Comment by hawkrives** [comment]
Didn't they set it on containerInfo in the forum though?

**Comment by rye** [comment]
Yes, but I'm not sure why that would be any different, necessarily.  I think the `docker` key is analogous to the `containerInfo` key, and to me it would make sense to have the `cmd` key be there too.

Also, it seems as though the increase in memory/CPU usage happens right when ~Flow~ the packager runs.

Looks like we just got a successful build! :tada: :tada: :tada:

**Comment by rye** [comment]
If you want, @hawkrives, I could merge this into ~the existing Gradle-related PR~ #2170 that was failing mysteriously, and we could see if this fixes that one. (In which case I think this should be our solution, perhaps.)

**Comment by hawkrives** [comment]
I'll take this over that PR!

**Comment by rye** [comment]
Okay! We can merge this once approved.  If it doesn't solve it, we can always revert back.

**Review by hawkrives — DISMISSED** [review]

**Comment by hawkrives** [comment]
>Actually, do you mind adding a comment to the yaml file?

Yeah, I can do that.  I also added `Closes` to the OP if that's okay with you.

**Comment by hawkrives** [comment]
Documenting why we have that?

**Comment by hawkrives** [comment]
I'm still approving with or without.

**Review by hawkrives — APPROVED** [review]

**Comment by rye** [comment]
Thanks, @hawkrives.  We'll see, if this build passes twice in a row I'll be very happy.

**Comment by hawkrives** [comment]
Yeah; I just looked at the queue log for master… there's a lot of red there. Hopefully that goes much greener now.

**Comment by rye** [comment]
It seems we have another successful Android build 😁.

**Comment by rye** [comment]
Here we go! Merging.

**Pull request merged**
